### DISCUSSION TITLE
Two Thermostats in one NSPanel

### DISCUSSION THREAD

**Discussion by DerT94** [Blackymas/NSPanel_HA_Blueprint#2031] [Q&A] [answered]
Hey,

is it possible to configure a second climate for an NSPanel.
I have a room with heating in which I don't want to install an NSPanel.
My idea is to configure a second thermostat on an NSPanel and use Relay2 for it.
If someone has a better idea I'll take that too :)

BR
Thomas

**Comment by edwardtfn** [comment]
It's probably easy to get there using customization, but having offline control of two climates would be quite a lot of work to do. So, if you are fine about having one of those able to change settings only when HA is available, this is probably possible.

**Reply by edwardtfn** [reply, marked as answer]
Thinking more about this... As you are controlling the thermostat for another room (let's call it "Room B"), you probably don't want to use the panel's temperature sensor anyways, so you will be relying on Home Assistant for temperature.
In this case, I would suggest you use the addon to have local control on Room A, where you can use the embedded temperature sensor, have all control local, etc. Then for Room B you set a [Generic Thermostat](https://www.home-assistant.io/integrations/generic_thermostat/) in Home Assistant, associating panel's relay 2 to the heater and a temperature sensor from Room B.
This will create a climate entity and you will be able to associate that to any button in your panel. 😉

**Reply by DerT94** [reply]
That's correct. I am using an DHT22 sensor for the room temperature.
I didn't know the Generic thermostat. Thank you :)
I will give it a try.
